dart_sys/src/tests.rs

```diff
@@ -0,0 +1,1698 @@
+#[allow(unused_imports)]
```

Comment:
we need to setup `ci.yaml` to test these as well (should be quite simple, just a few more lines)

Reply:
proposed `ci.yml` changes:
https://github.com/fzyzcjy/flutter_rust_bridge/pull/894/commits/5907df4f5cc48fde84d998f18a970997ffa18297#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133dd

Reply:
LGTM, roughly it is like that

Reply:
For the purposes of getting started on this project, I am going to close this review and then as we work on the `dart_sys/` crate we can update the tests are write better ones as well as test anything we add :)